client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -42,37 +42,37 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 		super( props );
 		const { translate } = props;
 		const legalTypes = {
-			ABO: translate( 'Aboriginal', {
-				comment: 'Refers to indigenous peoples, specifically of Canada.'
-			} ),
-			ASS: translate( 'Association (Unincorporated)', {
+			CCT: translate( 'Canadian Citizen' ),
+			CCO: translate( 'Canadian Corporation' ),
+			RES: translate( 'Permanent Resident' ),
+			GOV: translate( 'Government' ),
+			EDU: translate( 'Educational Institution' ),
+			ASS: translate( 'Unincorporated Association', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Association (Unincorporated)' )` *ES Score: 30*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).